website/pages/docs/nia/installation/install.mdx

```diff
@@ -40,6 +47,22 @@ Usage of consul-terraform-sync:
       Print the version of this daemon.
 ```
 
-# Connect your Network Device
+## Connect your Network Device
+
+Consul-Terraform-Sync interacts with your network device through a network driver. For the Terraform network driver, Consul-Terraform-Sync uses Terraform providers to make changes to your network infrastructure resources. You will need to [search for an existing](/docs/nia/installation/requirements#finding-and-using-terraform-providers) Terraform Provider for each of your network devices or [create a new one](/docs/nia/installation/requirements#how-to-create-a-provider).
+
+Once you have identified a Terraform provider for all of your network devices, you can configure them in Consul-Terraform-Sync with a [provider block](/docs/nia/installation/configuration#provider) for each network device. Below is an example:
+
+```hcl
+provider "fake-firewall" {
+  address = "10.10.10.10"
+  username = "admin"
+  password = "password123"
+}
+```
+
+This provider is then used by task(s) to execute a Terraform module that will update the related network device.
+
+### Multiple Instances per Provider
 
-## Multiple Instances per Provider
+You might have multiple instances of the same type of network device; for example, multiple instances of a firewall or load balancer. You can configure each instance with its own provider block and distinguish it by the `alias` field. See [multiple provider configurations](/docs/nia/installation/configuration#multiple-provider-configurations) for more details and an example of the configuration.
```

Comment:
Let's try to adhere closely to Terraform verbiage: field -> meta-argument ([source](https://www.terraform.io/docs/configuration/providers.html#alias-multiple-provider-configurations))